<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPresencial</class>
 <widget class="QMainWindow" name="ControlPresencial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>305</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>ControlPresencial</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>200</width>
      <height>67</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/es-resources/Resources/Logo.gif);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Digitalclock">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 19pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Clock</string>
    </property>
   </widget> 
   <widget class="QLabel" name="DateTime">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>65</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 13pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Datetime</string>
    </property>
   </widget> 
   <widget class="QLabel" name="LabelDispositivo"><!--
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Dispositivo</string>
    </property>-->
   </widget>
   <widget class="QLabel" name="LabelEdificio"><!--
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Edificio</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>-->
   </widget>
   <widget class="QLabel" name="LabelError">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LabelIP">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>9</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget> 
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
